ananse/peakpredictor.py

```diff
@@ -302,10 +306,16 @@ def _load_motifs(self, indirect=True, factors=None):
         # Create a graph of TFs where edges are determined by the Jaccard index
         # of the motifs that they bind to. For instance, when TF 1 binds motif
         # A and B and TF 2 binds motif B and C, the edge weight will be 0.33.
-        tmp_f2m = {}
-        if self.pfmfile is not None:
-            logger.debug("reading default file")
-            tmp_f2m = self._load_factor2motifs(indirect=True)
+        if indirect is False or factors is not None:
+            tmp_f2m = self._load_factor2motifs(
+                pfmfile=self.pfmfile, indirect=True, factors=None
```

Comment:
@JGASmits, when using a custom pfmfile, the Jaccard index needs to use the custom pfmfile too, right?

Reply:
It then needs both the costum pfm file, and the costum factors2motifs file i presume